demo/lego_official_projects/include/jquery.ui.touch-punch.min.js

```diff
@@ -1,11 +1,11 @@
 /*!
  * jQuery UI Touch Punch 0.2.3
  *
- * Copyright 2011–2014, Dave Furfero
+ * Copyright 2011-2014, Dave Furfero
```

Comment:
The original - above was not an ascii character, the web server was unable to serve jquery.ui.touch-punch.min.js due to it containing a non-ascii character.